### PR TITLE
Fix transactions

### PR DESCRIPTION
**Pull request by saltz**
TODO fix package.json layer

### CODE REVIEW

**Comment by saltz** [comment]
The latest build was marked as unstable due to the coverage requirement.

File: src/main.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { ConstantStore } from "./stores/constant.store";
 
 async function main() {
     try {
-        await initSentry();
```
Comment (timofudala) [inline src/main.ts:12]: We'd like to be notified of errors that can possibly occur whenever the peer is being initialized. We should, therefore, wait before sentry has been initialized.

File: src/services/concretes/state.service.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                const states: State[] = [];
+
+                for (const transaction of transactions) {
+                    let senderState = await this.dataAccessLayer
+                        .getStateAsync(transaction.transactionHeader.senderKey);
+                    let recipientState = await this.dataAccessLayer
+                        .getStateAsync(transaction.transactionHeader.recipientKey);
+
+                    senderState = this.updateStateAmount(
+                        "-",
```
Comment (timofudala) [inline src/services/concretes/state.service.ts]: I'd prefer you to change the string to an enum. You could consider keeping the enum privately within this class if that's possible.

File: src/stores/constant.store.ts [hunk 3]
```diff
@@ -52,5 +54,6 @@ export class ConstantStore {
         this.DB_CONNECTION_STRING = process.env.DB_CONNECTION_STRING || "";
         this.DB_NAME = process.env.DB_NAME || "";
         this.INITIAL_PEER = process.env.INITIAL_PEER || "";
+        this.DEFAULT_STAKE_AMOUNT = 1;
```
Comment (TimoF97) [inline src/stores/constant.store.ts]: There already exists a constant for this, it's name is rather odd though; `GENESIS_STAKE_AMOUNT`. We should probably change its name to `default` and remove your newly added const.

**Review by timofudala — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (saltz) [reply src/main.ts:12]: `initSentry()` is not an async function and thus should not be awaited. (This was also flagged by sonarqube)

**Review by timofudala — APPROVED** [review]

**Pull request merged**